.github/workflows/publish.yml

```
@@ -0,0 +1,38 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    tags:
+    # must align with the tag-pattern configured on pub.dev, often just replace
+    #  with [0-9]+.[0-9]+.[0-9]+*
+    - 'v[0-9]+.[0-9]+.[0-9]+*' # tag-pattern on pub.dev: 'v'
```

```suggestion
    - '[0-9]+.[0-9]+.[0-9]+*' # tag-pattern on pub.dev: ''
```
We don't use `v` in our tags